stories/topics/proc-azure-provisioning-aap.adoc

```diff
@@ -38,6 +38,10 @@ Keep this resource group isolated from other resource groups, including the _Res
 
 The application will begin provisioning.
 
+You can use the deployment engine to view the progress of your deployment approximately 10 minutes after you begin provisioning.
```

Comment:
I prefer to say "You can use the deployment engine to view the progress of your deployment approximately a few minutes after Azure web shows "Your deployment is complete". During my testing, if i tried to access the Deployment Engine UI immediately after Azure web shows "Your deployment is complete", I would get "the site is not accessible". The deployment engine URL was becoming accessible after I waited for a few minutes.